contrib/mesos/pkg/scheduler/plugin.go

```diff
@@ -448,8 +453,8 @@ func (q *queuer) Run(done <-chan struct{}) {
 			} else {
 				// use ReplaceExisting because we are always pushing the latest state
 				now := time.Now()
-				pod.deadline = &now
-				if q.podQueue.Offer(pod, queue.ReplaceExisting) {
+				pod.eventTime = &now
```

Comment:
ok

On Mon, Sep 28, 2015 at 7:07 PM, Karl Isenberg <notifications@github.com>
wrote:

> In contrib/mesos/pkg/scheduler/plugin.go
> <https://github.com/kubernetes/kubernetes/pull/14610#discussion_r40617160>
> :
>
> > @@ -448,8 +453,8 @@ func (q *queuer) Run(done <-chan struct{}) {
> >  			} else {
> >  				// use ReplaceExisting because we are always pushing the latest state
> >  				now := time.Now()
> > -				pod.deadline = &now
> > -				if q.podQueue.Offer(pod, queue.ReplaceExisting) {
> > +				pod.eventTime = &now
>
> The default clock impl delegates to time, so it's not a big deal unless
> we want to use FakeClock to test the scheduler plugin itself. At that point
> the clock injection would need to be moved up a level.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/14610/files#r40617160>.
>